docs/web3.eth.rst

```diff
@@ -955,14 +955,19 @@ The following methods are available on the ``web3.eth`` namespace.
 
     .. code-block:: python
 
-        >>> web3.eth.estimateGas({'to': '0xd3CdA913deB6f67967B99D67aCDFa1712C293601', 'from': web3.eth.coinbase, 'value': 12345})
+        >>> web3.eth.estimate_gas({'to': '0xd3CdA913deB6f67967B99D67aCDFa1712C293601', 'from':
+        web3.eth.coinbase, 'value': 12345})
```

Comment:
I think this reads better on the same line. You can check out what the docs look like by running `make docs`